### ISSUE TITLE
training data queries

### ISSUE THREAD

**Issue by EvelyneCalista** [cwmok/C2FViT#12]
Hi, thanks for your great work.

I have downloaded the OASIS data from your provided link. However, for the semi supervised and unsupervised template matching require some files for template matching and pairwise which i couldn't find on the provided data folder. 
_semi supervised pairwise: seg35_onehot.nii.gz
semi supervised template matching: seg4_mni_onehot.nii.gz
semi supervised template matching: MNI-maxprob-thr50-1mm_pad_RSP_oasis_onehot.nii.gz
unsupervised template matching : seg4_mni.nii.gz_

Could you please help me to explain how can i get these kinds of data?
Thank you so much!!

**Comment by cwmok** [comment]
Hi @EvelyneCalista,

Thanks for your interest in our work.
The data you mentioned is just the one-hot encoding of the label files. See the example data: https://github.com/cwmok/C2FViT/tree/main/Data

I will attach my preprocessing script below. Note that I currently don't have time to clean the code. The attached code serves as a reference only.

>> unsupervised template matching : seg4_mni.nii.gz

This should be the segmentation map of MNI152 template but with only the selected 4 anatomical structures.

**Comment by cwmok** [comment]
Here is the preprocessing code I use to get the one-hot encoding label:
[preprocessing_script.zip](https://github.com/cwmok/C2FViT/files/13403134/preprocessing_script.zip)

**Comment by EvelyneCalista** [comment]
Hi @cwmok 

Thank you for the reference code. 
I would like to reconfirm that for this file 'seg4_mni.nii.gz' , do the input file is from 'seg35.nii.gz' like in your reference code?

**Comment by cwmok** [comment]
Hi @EvelyneCalista ,

Yes, you're correct. Please see "preprocess_MNI152.py" for more details.

**Comment by EvelyneCalista** [comment]
Hi @cwmok ,

Got it, I can run the training code.
Thank you for your help! :)
